runtime/doc/news.txt

```diff
@@ -307,8 +307,8 @@ UI
   |hl-PmenuSel| and |hl-PmenuMatch| both inherit from |hl-Pmenu|, and
   |hl-PmenuMatchSel| inherits highlights from both |hl-PmenuSel| and
   |hl-PmenuMatch|.
-• |vim.diagnostic.setqflist()| updates existing diagnostics quickfix list if one
-  exists.
```

Comment:
Is this appropriate? Given that the previous feature never made it into a release, should I just coalesce these 2 features into a single note?